docs/source/devguide/dev_assisted_package_creation.rst

```diff
@@ -0,0 +1,104 @@
+ .. dropdown:: Distribution Statement
+
+    | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+    | # # #
+    | # # # Author:
+    | # # # Naval Research Laboratory, Marine Meteorology Division
+    | # # #
+    | # # # This program is free software: you can redistribute it and/or modify it under
+    | # # # the terms of the NRLMMD License included with this program. This program is
+    | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    | # # # for more details. If you did not receive the license, for more information see:
+    | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Developer Assisted Package Creation
+===================================
+
+Why We Assist With Package Creation
+-----------------------------------
+
+The GeoIPS Development team is committed to integrating a variety of geospatial
+processing routines into GeoIPS, or as a standalone packages that can interact with
+GeoIPS. Specifically we developers at CIRA are tasked with integrating proprietary
+software developed by researchers to work alongside GeoIPS. If you have access, see the
```

Comment:
Can we link a public package?

Reply:
All of the packages I've implemented are private.

Reply:
@jsolbrig Do you know of any public ones that we can offer as an example?